client/components/credit-card/docs/example.jsx

```diff
@@ -0,0 +1,44 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React, { Component, Fragment } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import CreditCard from '..';
+
+class CreditCards extends Component {
+	static displayName = 'CreditCard';
+
+	state = {
+		cards: [
+			{
+				card: '4242',
+				card_type: 'visa',
```

Comment:
these really ought to be camelCase as we have standard in Calypso.

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/24485/commits/646bea9c3b0278b73c80ce0cf34253645391eb02